lib/std/child_process.zig

```diff
@@ -156,6 +156,8 @@ pub const ChildProcess = struct {
         Ignore,
         Pipe,
         Close,
+        // Pass an open file to the process. This will take ownership of the file, so **do not close it.**
+        File,
```

Comment:
this is really just a nit, but the other entries don't have comments, and it seems out of place. maybe put the comment "in front" of the entry.

```zig
 File, // Pass an open file to the process. This will take ownership of the file, so **do not close it.**
```
if you meant for it to show up when you hover over it, a docstring is done with `///`, and it does require to be on a separate line.

this is really minor 😄, just wondering.

Reply:
I had it there out of habit. I moved it to the right of the variant to make it clear that it's more of a note than a doc string. Thank you for all of your help!